Documentation/upgrade.md

```diff
@@ -40,6 +40,7 @@ Review the [health verification section](#health-verification) in order to verif
 This persists metadata on host nodes, enabling pods to be terminated during the upgrade and for new pods to be created in their place.
 More details about `dataDirHostPath` can be found in the [Cluster CRD readme](./ceph-cluster-crd.md#cluster-settings).
 * All pods consuming Rook storage should be created, running, and in a steady state.  No Rook persistent volumes should be in the act of being created or deleted.
+* All PVC creation should be complete prior to starting the upgrade. If you have PVCs that are stuck in pending state, it is safe to delete them, verify both PV and PVC are deleted and continue with the upgrade.
```

Comment:
being in the pending state means that they have not yet been created and there would be no data that gets deleted from this operation, right?  then the user can just recreate them (successfully) after the upgrade is done?

Reply:
I am referring to the one where the pvc entry has been made but we are unable to complete the request. In that scenario, we will have a bunch of pvcs in creating state and stuck there until we delete them. Maybe changing the text to say that would be better?

Reply:
i thought about this a little more, I think we should be more explicit about what's happening here and what their options are.  Since the fix for #2072 that is included in the new 0.8.2 operator that the user is upgrading to should automatically fix this issue without the user having to do anything, I think we should give them that option still here if they are comfortable with that.  Less manual steps is better.

I suggest we come up with prose to touch on the following:
* make this info a sub bullet under the one above about "no PVs should be in the act of being created", so that it is more in context
* due to #2072, some PVCs may be in the pending state, upgrading the operator *can* automatically fix this
* the user has the option of deleting the pending PVCs and recreating them manually after the upgrade, OR they can just upgrade the operator and check the pending PVC status to verify if they get fixed and transition to the bound status

Does that make sense? let me know if i have misunderstood any of this issue.